### PR TITLE
Add dockerspawner

### PR DESCRIPTION
**Pull request by dhirschfeld**

### CODE REVIEW

**Comment by dhirschfeld** [comment]
@minrk - let me know if you'd like to be added as a maintainer for this recipe

**Comment by mariusvniekerk** [comment]
you're probably missing dependencies

File: recipes/dockerspawner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ac7a8275dd33a73e31da9d3d13849da579f0f248377d5fa558d1ceb793190a6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
```
Comment (mariusvniekerk) [inline recipes/dockerspawner/meta.yaml]: this likely needs to have --no-deps in it

Comment (dhirschfeld) [reply recipes/dockerspawner/meta.yaml]: yep - found that. Why did it change?

Doesn't it also need `--ignored-installed` and `--no-cache-dir`?

**Comment by dhirschfeld** [comment]
The deps look good AFAICS:

https://github.com/jupyterhub/dockerspawner/blob/10d3a373370de4d1edfb6e9a092524d2ff768f31/requirements.txt#L1-L3

**Comment by minrk** [comment]
@dhirschfeld sure, feel free to add me

**Comment by dhirschfeld** [comment]
I think this is good to go @mariusvniekerk!

**Comment by dhirschfeld** [comment]
Thanks! :)

**Pull request merged**